yara-x/src/tests/mod.rs

```diff
@@ -3099,7 +3100,7 @@ fn test_macho_module() {
             macho.dylibs[1].timestamp == 2
     }
     "#,
-        &MACHO_X86_64_DYLIB_FILE
+        &dylib_data
```

Comment:
Same for all bellow:
There is probably no need to have this also here as it is just double testing the same fields that are already tested in `test_modules` framework. The only things that can't be tested there and is better to include here are:

- constants
- exported functions

Sure there is no problem having it also here, but in long-term scenario it will just increase the code and places to maintain in case something changes (instead of just regenerating goldenfiles).

Reply:
ah ok I see now, the duplicate tests have been removed :smile: